dbt_invoke/properties.py

```diff
@@ -825,12 +837,33 @@ def _structure_property_file_dict(location, resource_dict, columns_list):
         column_dict = existing_columns_dict.get(
             column, _get_property_column(column)
         )
+        # If dbt_invoke_template.yml exists, adds templated properties to column 
+        # when not already present
+        if template_yml and 'columns' in template_yml.get(resource_type):
```

Comment:
```suggestion
        if template_yml and 'columns' in template_yml.get(resource_type, dict()):
```
If `resource_type` is not in `template_yml`, then `template_yml.get(resource_type)` returns `None`, which would result in `'columns' in None` and cause an error.